.github/workflows/check.yml

```
@@ -38,10 +35,10 @@ jobs:
           key: ${{ runner.os }}-android-sdk-${{ hashFiles('**/android/gradle/wrapper/gradle-wrapper.properties') }}
           restore-keys: ${{ runner.os }}-android-sdk-
       - run: echo '${{ secrets.ENCODED_RELEASE_KEYSTORE }}' | base64 --decode > android/playstore.keystore
+      - name: Add Flutter to PATH
+        run: echo "/home/ubuntu/flutter/bin" >> $GITHUB_PATH
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:36: Double quote to prevent globbing and word splitting

(shellcheck)